data/amd/register/msr/MSRC001_0055.yml

```diff
@@ -0,0 +1,22 @@
+- name: MSRC001_0055
+  long_name: "Reserved"
+  purpose: |
+       "
+        MSRC001_0055 [Reserved.] (Core::X86::Msr::IntPend)
+       "
+  size: 64
+  arch: amd64
+
+  access_mechanisms:
+      - name: rdmsr
+        is_read: True
+        address: 0xC0010055
+
+      - name: wrmsr
+        is_write: True
+        address: 0xC0010055 
+
+  fieldsets:
+      - name: latest
+        condition: "Fieldset valid on latest version of the AMD architecture"
+        size: 64
```

Comment:
You technically skipped all of the reserved fields in all of the MSRs, but this one looks like it really needs a single field defined in this fieldset:

```
- name: Reserved
  long_name: "Reserved"
  lsb: 0
  msb: 63
  readable: True
```